app/views/jobs/_form.html.erb

```diff
@@ -24,14 +24,13 @@
     <div class="col">
       <div class="form-group">
         <label for="description">Art der Anstellung *</label>
-        <%= text_field_tag(
-              "job_payload_employment_type",
-              (@job.payload || {})["employmentType"],
-              required: true,
-              placeholder: "Vollzeit, Teilzeit, Ausbildung, ...",
-              class: "form-control",
-              name: "job[payload][employment_type]"
-            )%>
+        <% @categories = @categories.select { |category| category.tag_list.split(",").include?("generic_item_job") } %>%
+        <%= select_tag(
+                        "job[payload][employment_type]",
+                        options_for_select(@categories.map(&:name), (@job.payload || {})["employmentType"]),
+                        include_blank: true,
+                        class: "form-control"
+                      ) %>
```

Comment:
`required` is missing now?

Reply:
Required wieder hinzugefügt:

<img width="708" alt="Bildschirmfoto 2024-05-02 um 16 06 57" src="https://github.com/smart-village-solutions/smart-village-app-cms/assets/90779/10a30b81-e075-45a8-8754-8289649c3b4b">